drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -476,6 +478,24 @@ func (c *pvcController) getPVCControllerDeploymentSpec(
 						},
 					},
 					Affinity: &v1.Affinity{
+						PodAffinity: &v1.PodAffinity{
+							RequiredDuringSchedulingIgnoredDuringExecution: []v1.PodAffinityTerm{
+								{
+									TopologyKey: "kubernetes.io/hostname",
+									LabelSelector: &metav1.LabelSelector{
+										MatchExpressions: []metav1.LabelSelectorRequirement{
+											{
+												Key:      "storage",
```

Comment:
Let's use the key and value constants from https://github.com/libopenstorage/operator/blob/master/pkg/constants/metadata.go#L11 to be consistent.